.github/workflows/trigger-build-release.yml

```
@@ -0,0 +1,46 @@
+name: Trigger Build Release
+
+on:
+  release:
+    types: [published]
+  workflow_dispatch:
+    inputs:
+      version:
+        description: Version of Valkey to build
+        required: true
+      environment:
+        description: Environment to build
+        required: false
```

Lets make this required and add a dropdown
```suggestion
      environment:
        description: Environment to build
        required: true
        type: choice
        options:
        - dev
        - prod
```